.changeset/tabaxi-fellas-inspire.md

```diff
@@ -0,0 +1,141 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+TableSchema (alpha) APIs have been added
+
+Users may now represent dynamic, tabular data using Shared Tree.
+New APIs have been added under a `TableSchema` namespace for creating column, row, and table node schema.
```

Comment:
They could do this before. This change provides a standard easy way to do it. I think something liek this would be better:

```suggestion
A `TableSchema` utility has been added to Shared Tree for managing dynamic, tabular data.
This new `TableSchema` namespace contains APIs for creating column, row, and table [node schema](https://fluidframework.com/docs/api/fluid-framework/treenodeschema-typealias).
```